plugins/@grouparoo/demo/src/configFiles.ts

```diff
@@ -64,3 +69,28 @@ function updatePurchases(configDir) {
   const out = JSON.stringify(contents);
   fs.writeFileSync(appPath, out);
 }
+
+async function unlockAll() {
+  // unlock these for the demo so they can be shared
+  const models = api.sequelize.models;
+  for (const name in models) {
+    const Model = models[name];
+    // if (["Team"].includes(name)) {
+    //   continue;
+    // }
+    const attributes = Model.rawAttributes;
+    if (attributes.locked) {
+      log(3, `Unlocking ${name}`);
```

Comment:
probably want to remove this log?

Reply:
This is the demo system log that only logs when they want it more verbose.